modules/base/rendering/renderablesphere.cpp

```diff
@@ -244,6 +268,38 @@ RenderableSphere::RenderableSphere(const ghoul::Dictionary& dictionary)
     addProperty(_disableDepth);
 
     setBoundingSphere(_size);
+
+    if (p.colorMap.has_value()) {
+        _transferFunctionPath = p.colorMap->string();
```

Comment:
Since the other varialbe is "is using color map", please rename this property to "color map" rather than "transfer function" :)